src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -213,6 +216,28 @@ export default class GroupComparisonPage extends React.Component<
                             <AlterationEnrichments store={this.store} />
                         </MSKTab>
                     )}
+                    {
+                        <MSKTab
+                            id={GroupComparisonTab.PATHWAYS}
+                            linkText={'Pathways'}
+                            anchorClassName={
+                                this.store.alterationsTabUnavailable
```

Comment:
use the classNames package to apply classes conditionally.  just search project for "classnames" and you'll find example